src/application/wizard/CreateApplicationPage.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import { RocketIcon } from '@patternfly/react-icons';
+
+import {
+  Bullseye,
+  Button,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Form,
+  InputGroup,
+  TextInput,
+  Title,
+} from '@patternfly/react-core';
+import { UpsClientFactory } from '../../utils/UpsClientFactory';
+
+interface State {
+  appName: string;
+}
+
+interface Props {
+  onFinished: () => void;
+}
+
+export class CreateApplicationPage extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      appName: '',
+    };
+  }
+
+  private readonly createApp = async (name: string) => {
+    try {
+      await UpsClientFactory.getUpsClient().applications.create(name);
+      this.props.onFinished();
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  render(): React.ReactNode {
+    return (
+      <EmptyState variant={EmptyStateVariant.full}>
+        <EmptyStateIcon icon={RocketIcon} />
+        <Title headingLevel="h5" size="lg">
+          Create your an application
+        </Title>
+        <EmptyStateBody>
```

Comment:
This looks like its being pushed to the right by the "create your first application"  component on the left.

Reply:
That's on purpose: it is a [wizard](https://www.patternfly.org/v4/documentation/react/components/wizard) This PR is about the first page of the wizard, in future PRs we will add other pages.